recipes/rapidyaml/all/test_package/CMakeLists.txt

```diff
@@ -6,7 +6,9 @@ conan_basic_setup(TARGETS)
 
 find_package(ryml REQUIRED CONFIG)
 
-set(CMAKE_CXX_STANDARD 11)
-
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} ryml::ryml)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

Comment:
you should update `cmake_minimum_required` to 3.8

Reply:
Thank you.
I tend to forget cxx_std_11 added since 3.8.